examples/components/PNEUMONIA/traininsilo/run.py

```diff
@@ -430,6 +434,7 @@ def get_arg_parser(parser=None):
         required=False,
         help="Total number of epochs for local training.",
     )
+    parser.add_argument("--batch_size", type=int, required=False, help="Batch Size")
```

Comment:
you will have to set default=32 here, or else I believe you'll use None as default value, and your code might fail line 134 when None is given as value for batch_size to DataLoader

Reply:
Absolutely, that makes sense. Just one thing to confirm -- it looks like the "batch_size" as we implemented in all examples is actually a required parameter, otherwise we will get key error in the submit.py saying that the "batch_size" does not exist in the config. Please correct me if that is wrong.🙂

Reply:
yes it's required, so somehow this required=True here is redundant, but if at some point we change the definition somewhere else, I don't want this script to fail. Thx for fixing it!